dpgen2/op/run_nvnmd_train.py

```
@@ -0,0 +1,670 @@
+            ),
+            Argument(
+                "train_args",
+                str,
+                optional=True,
+                default="",
+                doc=doc_train_args,
+            ),
+        ]
+
+    @staticmethod
+    def normalize_config(data={}):
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)